js/menu.js

```diff
@@ -321,10 +321,14 @@ function Menu() {
 
 Menu.prototype.documentKeydown = function (e) {
   e.stopPropagation();
-  if (e.keyCode === 80) {
+  if (e.key === 'p') {
     this.togglePinEntry();
-  } else if (e.keyCode >= 48 && e.keyCode < 58) {
+  } else if ([0, 1, 2, 3, 4, 5, 6, 7, 8, 9].includes(parseInt(e.key))) {
     this.selectPin((e.keyCode - 9) % 10);
+  } else if (e.key === '`') {
+    if (document.location.hash) {
+      document.querySelector(document.location.hash).scrollIntoView(true);
```

Comment:
This could also be done with `document.location = document.location`, but that felt gross to me.